README.md

```diff
@@ -304,6 +304,34 @@ Responses:
 
 `:id` Is the id of an item in the feed.
 
+### `GET /api/mood`
+
+> Get list of day by day mood
+
+Query parameters:
+
+* `cityId` Integer. If specified, returned ratingCity is for the given city.
+* `cityName` String. If specified, returned ratingCity is for the given city.
+* `teamId` Integer. If specified, returned ratingTeam is for the given team.
+* `teamName` String. If specified, returned ratingTeam is for the given team.
```

Comment:
I think the behaviour is a bit odd now. Is it so that it *always* returns the mood for the user, city and team? And if city and/or team has been specified, it returns the also for the user, city and team? If so, I think this could work as follows:

1. If no query params are given, return the mood for the user, user's team and user's city.
2. If query params are given, return *only* those moods that are requested with query params.